src/modules/updateTaskData.js

```diff
@@ -0,0 +1,17 @@
+import updateLocalStorage from './updateLocalStorage.js';
+import completedCounter from './completedCounter.js';
+
+const updateTaskDescription = (index, value) => {
+  const taskList = JSON.parse(localStorage.getItem('taskListData'));
+  taskList[index - 1].description = value;
+  updateLocalStorage(taskList);
+};
+
+const updateTaskCompleted = (index, boolean) => {
+  const taskList = JSON.parse(localStorage.getItem('taskListData'));
+  taskList[index - 1].completed = boolean;
+  updateLocalStorage(taskList);
+  completedCounter();
+};
```

Comment:
Good use of localStorage to keep your data. Maybe you could create just one function here with three arguments (index,value,boolean) instead of two, the result will be the same and you can save a few lines of code, this will avoid you repeating unnecessary lines of code and better management if you have to change the code in the future